srcpkgs/freeciv/template

```diff
@@ -10,12 +10,14 @@ makedepends="SDL2_gfx-devel SDL2_image-devel SDL2_mixer-devel SDL2_ttf-devel
 short_desc="Free and Open Source empire-building strategy game - server/data"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-or-later"
-homepage="http://freeciv.org/"
-changelog="https://freeciv.fandom.com/wiki/NEWS-${version}"
+homepage="https://www.freeciv.org/"
+changelog="https://www.freeciv.org/wiki/NEWS-${version}"
```

Comment:
This URL redirects to the fandom.com URL. Why the change?

Reply:
"https://www.freeciv.org/wiki/" is the official URL that does not make the assumption of the wiki being located on wikia. Maybe freeciv project won't be using wikia forever (and that URL worked already before migrating wiki to wikia)